modules/optimization/examples/bimaterial/pest_instruction.ins

```diff
@@ -0,0 +1,6 @@
+pif @
```

Comment:
do we need these files? .ins, .tpl and .pst

if not -> gitignore?

Reply:
removed this example

Reply:
no, these files were from using a different optimization code for comparison.  But we don't need to do that.